apidef/oas/middleware.go

```diff
@@ -1035,7 +1035,7 @@ type VirtualEndpoint struct {
 	Name string `bson:"name" json:"name"` // required.
 	// Path is the path to js file.
 	Path string `bson:"path,omitempty" json:"path,omitempty"`
-	// Body is the js function to execute encoded in base64 format.
+	// Body represents the js function body content. It is stored in base64 format.
```

Comment:
The previous description sounds clearer to me

Reply:
This change is based on comment https://github.com/TykTechnologies/tyk-docs/pull/2330#discussion_r1121362561. 
If we decide to go without making this change, we can discard this PR and get tyk-docs PR merged.

Reply:
it still needs a comma or full stop

Reply:
yes

Reply:
but we need to add this `if path and body are present, body would take precedence`